### PR TITLE
Added decorator `skipRocmIfTorchInductor`

### PR DESCRIPTION
**Pull request by pragupta**
This commit is already in Upstream, but pulling it here as we have inductor CI enabled and need this decorator to skip any unwanted tests.

Upstream PR: https://github.com/pytorch/pytorch/pull/107760

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     TEST_WITH_TORCHINDUCTOR, TestCase, TEST_WITH_ROCM, run_tests, IS_JETSON,
     IS_WINDOWS, IS_FILESYSTEM_UTF8_ENCODING, NO_MULTIPROCESSING_SPAWN,
     IS_SANDCASTLE, IS_FBCODE, IS_REMOTE_GPU, skipIfTorchInductor, load_tests, slowTest, slowTestIf,
-    TEST_WITH_CROSSREF, skipIfTorchDynamo, set_default_dtype,
+    TEST_WITH_CROSSREF, skipIfTorchDynamo, skipRocmIfTorchInductor,
```
Comment (pruthvistony) [inline test/test_torch.py:35]: Why is set_default_dtype removed?

**Review by pruthvistony — CHANGES_REQUESTED** [review]

File: test/test_torch.py (hunk 1 above)
Comment (pragupta) [reply test/test_torch.py]: Sorry, bad merge

**Review by jataylo — APPROVED** [review]
LGTM but could we link the upstream PR in the description @pragupta

**Comment by pragupta** [comment]
> LGTM but could we link the upstream PR in the description @pragupta

Updated PR description.

**Review by pruthvistony — APPROVED** [review]

**Pull request merged**